core/server/master/src/main/java/alluxio/master/file/replication/ReplicationChecker.java

```diff
@@ -219,6 +220,12 @@ private void check(Set<Long> inodes, ReplicationHandler handler, Mode mode) {
         LOG.warn("Unable to complete the replication check: {}, will retry later.",
             e.getMessage());
         return;
+      } catch (ResourceExhaustedException e) {
+        LOG.warn("The job service is busy, will retry later: {}", e.getMessage());
+        LOG.debug("Exception: ", e);
```

Comment:
I prefer doing an `if (LOG.debugEnabled())` check instead of logging two related messages separately.

Reply:
Will be removed once combined with `JobDoesNotExist` exception in next commit